linera-service/src/server.rs

```diff
@@ -343,6 +344,10 @@ enum ServerCommand {
         #[arg(long)]
         wasm_runtime: Option<WasmRuntime>,
 
+        /// The maximal number of simultaneous chains in memory handling requests.
+        #[arg(long, default_value = "400")]
```

Comment:
I'd rather write: "The maximal number of chains loaded in memory at a give time."

Actually, "active" is misleading. Should this be `max_loaded_chains` everywhere?

Reply:
I believe this value is not about CPU usage but more about the memory available. We should be able to set reasonable defaults.

Reply:
True, but I think we don't know how much memory each chain worker uses :/ I think it depends on the loaded applications and how much state they store?

Reply:
Indeed we should probably start thinking about more fine-grained resource management.